src/Rings/mpoly-localization_types.jl

```diff
@@ -178,6 +178,7 @@ mutable struct MPolyComplementOfKPointIdeal{
     length(a) == ngens(R) || error("the number of variables in the ring does not coincide with the number of coordinates")
     n = length(a)
     kk = coefficient_ring(R)
+    @assert kk isa Field "This localization is currently only available over fields"
```

Comment:
```suggestion
    @req kk isa Field "This localization is currently only available over fields"
```
`@req` will always check the condition, while `@assert` may get optimized away by julia. so better to use `@req` in input sanitation cases like here

Reply:
done